test/github/kv_test.rb

```diff
@@ -166,6 +166,15 @@ def test_setnx_overwrites_expired_key
     assert_equal "bar2", @kv.get("foo").value!
   end
 
+  def test_ttl
```

Comment:
Another test to test that a key with an expired ttl returns nil would likely also help with regressions. I can add it after merge though.